src/interfaces/Ticket.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Ticket {
```

Comment:
Nice separation of interface! However, I personally like the pattern to couple the interface with the closes feature/component. It's pretty much the same thing I posted with Node.js convention guide, but I think for front-end with a typescript pattern can follow such a pattern. Doesn't need to be so strict, but it would make sense to have type folder closely coupled with where it's supposed to be used to make the relationship between types and the consumer of it.

for further detail:
https://gist.github.com/menduz/f56c2d9aafc947aac09c4dc6a4042741#file-guide-md